ipld/encoding/Cargo.toml

```diff
@@ -9,16 +9,15 @@ repository = "https://github.com/filecoin-project/ref-fvm"
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
-serde_ipld_dagcbor = "0.2.2"
+serde_ipld_dagcbor = "=0.2.0"
 serde_tuple = "0.5"
 serde_repr = "0.1"
 cid = { version = "0.8.5", default-features = false, features = ["serde-codec", "std"] }
 thiserror = "1.0"
 anyhow = "1.0.56"
-fvm_ipld_blockstore = { version = "0.1", path = "../blockstore" }
 # multihash is also re-exported by `cid`. Having `multihash` here as a
 # depdendency is needed to enable the features of the re-export.
-multihash = { version = "0.16.1", default-features = false, features = ["blake2b", "multihash-impl"] }
+multihash = { version = "0.16.1", default-features = false }
```

Comment:
Since the previous code is the same as what's in `shared/Cargo.toml` now, this change isn't necessary.